src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/virtualusers/StackVirtualUsersFormula.kt

```diff
@@ -89,9 +89,11 @@ class StackVirtualUsersFormula private constructor(
     ): ProvisionedVirtualUsers<SshVirtualUsers> {
         logger.debug("Setting up $name...")
         val network = overriddenNetwork ?: NetworkFormula(investment, aws).provision()
+        val cloudformationTemplate = readResourceText("aws/virtual-users.yaml")
+        logger.info("template: $cloudformationTemplate")
```

Comment:
What is it for? Isn't it the same as reading from source code directly?

Reply:
debug, removing